pinot-query-planner/src/main/java/org/apache/calcite/sql/fun/PinotOperatorTable.java

```diff
@@ -207,6 +222,49 @@ public final void initNoDuplicate() {
     }
   }
 
+  /**
+   * This registers transform functions defined in {@link org.apache.pinot.common.function.TransformFunctionType}
+   * which are multistage enabled.
+   */
+  private void initTransformFunctions() {
+    // Walk through all the Pinot transform types and register those that are supported in multistage.
+    for (TransformFunctionType transformFunctionType : TransformFunctionType.values()) {
+      if (transformFunctionType.getSqlKind() == null) {
+        // Skip registering functions which are standard Calcite functions and functions which are not yet supported.
+        continue;
+      }
+
+      // Register the aggregation function with Calcite along with all alternative names
+      List<PinotSqlTransformFunction> sqlTransformFunctions = new ArrayList<>();
```

Comment:
this list is not necessary, you can simply register this one-by-one

Reply:
done.